frontend/packages/ceph-storage-plugin/src/components/modals/block-pool-modal/delete-block-pool-modal.tsx

```diff
@@ -85,8 +87,15 @@ const DeleteBlockPoolModal = withHandlePromise((props: DeleteBlockPoolModalProps
   }, [scResources, scLoaded, pvcResources, pvcLoaded, state.poolStatus, poolName, t]);
 
   // Delete block pool
+  const location = useLocation();
   const deletePool = () => {
-    handlePromise(k8sKill(CephBlockPoolModel, blockPoolConfig), () => close());
+    handlePromise(k8sKill(CephBlockPoolModel, blockPoolConfig), () => {
+      close();
+      // Go to block pool list page if user is currently on the details page
```

Comment:
Does it work from all the pages where we can delete the block pool ?

Reply:
Comment mentions only the details page

Reply:
I've tested this out based on https://github.com/openshift/console/pull/9514#discussion_r710995550. Should I remove the comment here (in case it's confusing)?

Reply:
```suggestion
      // Go to block pool list page if pool is deleted
```

I think this should be enough :)

Reply:
we are deleting the pool from two places:
    * list page
    * details page
    only the details page needs to redirect to the list page. 
   If you have a plan to backport this to OCP 4.9 branch then make sure this works when users use OCS 4.8 + OCP 4.9 from  ocs operator page as well.